assets/templates/credit_management/transaction_add.html

```diff
@@ -5,8 +5,29 @@
         <h3>Create new transaction</h3>
         <form method="post">
             {% csrf_token %}
-            {% include "snippets/bootstrap_form.html" with form=slot_form horizontal=True %}
-            <input type="submit" value="Create transaction" class="btn btn-block btn-primary"/>
+            {% include "snippets/bootstrap_form.html" with horizontal=True %}
+
+            <div class="row">
+                <div class="col-md-4 mt-1">
+                    <a class="btn btn-outline-primary btn-block"
+                       href="{{ redirect }}">
+                        Cancel
+                    </a>
+                </div>
+                {% if not chain %}
+                    <div class="col-md-8 mt-1">
+                        <input type="submit" value="Create" class="btn btn-block btn-primary"/>
+                    </div>
+                {% else %}
+                    <div class="col-md-5 mt-1">
+                        <input type="submit" value="Create" class="btn btn-block btn-primary"/>
+                    </div>
+                    <div class="col-md-3 mt-1">
+                        <input type="submit" value="Create More" name="add_another" class="btn btn-block btn-secondary"/>
```

Comment:
I haven't run this locally, but by looking at this file I don't think it's clear for a user what happens when she clicks on 'Create More'. Shouldn't it be something like 'Create and add another'?